Base/QTGUI/qSlicerModuleSelectorToolBar.cxx

```diff
@@ -187,6 +189,9 @@ void qSlicerModuleSelectorToolBarPrivate::init()
   QObject::connect(q, SIGNAL(toolButtonStyleChanged(Qt::ToolButtonStyle)), this->NextButton, SLOT(setToolButtonStyle(Qt::ToolButtonStyle)));
   this->NextButton->setEnabled(this->NextHistoryMenu->actions().size() > 0);
   this->NextButton->setShortcut(QKeySequence(Qt::ControlModifier + Qt::Key_Right));
+
+  // Install event filter to handle mouse back/forward buttons globally
+  qApp->installEventFilter(q);
```

Comment:
I'm not sure why `ViewExtensionsManagerAction` is inserted using a loop over child objects, perhaps that preferred over installing event filters over the entire app? For example, binding the shortcuts this way still alters the module selection even when a pop-up window is in focus and not the main windows. E.g. press Ctrl+F to bring up the Module finder pop-up, then press Ctrl+Left or Right. This action while the main window is not in focus is perhaps not expected. A web browser wouldn't do that if a file-picker pop-up was in focus.